libraries/Microsoft.Bot.Builder.LanguageGeneration/Templates.cs

```diff
@@ -133,16 +134,17 @@ public class Templates : List<Template>
         public IList<string> Options { get; set; }
 
         /// <summary>
-        /// Gets a value indicating whether lG parser/checker/evaluate strict mode.
-        /// If strict mode is on, expression would throw exception instead of return
-        /// null or make the condition failed.
+        /// Gets a value LG parser/checker/evaluate options.
         /// </summary>
         /// <value>
-        /// A value indicating whether lG parser/checker/evaluate strict mode.
         /// If strict mode is on, expression would throw exception instead of return
         /// null or make the condition failed.
+        /// replaceNull works all the time. 
```

Comment:
>works all the time [](start = 24, length = 18)

What does that mean?

Reply:
@cosmicshuai , we should make the description clear and accurate. 

first, this is not the place you explain each individual option, please remove this from here, and explain in the EvaluateOptions class on each property's comment

Reply:
description modified